kong/runloop/handler.lua

```diff
@@ -1323,8 +1323,8 @@ return {
 
       local ok, err, errcode = balancer_execute(ctx)
       if not ok then
-        local body = utils.get_default_exit_body(errcode, err)
-        return kong.response.exit(errcode, body)
+        local body = utils.get_default_exit_message(errcode, err)
+        return kong.response.error(errcode, body)
```

Comment:
Same here, let's check if .error already finds the default message.

Reply:
Same answer